common/src/main/resources/less/style.less

```diff
@@ -130,8 +130,25 @@ body {
   overflow-y: scroll;
 }
 
+// Inner area of the tree
+.tree-body-inner {
+}
+ 
 .rgl-body {
   height: 100%;
   position: absolute;
   top: 0px;
 }
+
+.obs-tree {
+  width: 100%;
+}
+
+.obs-tree-item {
+  margin-left: 25px;
+  padding: 5px;
+}
+
+.obs-badge {
+  width: 100% !important;
```

Comment:
It would be nice to avoid `!important` maybe we need a more specific selector

Reply:
It overrides the `width` set within Semantic UI's `Card`. We can pass a `^.width` to the `Card` within the code and it will take it, but if we want to set it via CSS, it wasn't being prioritized without the `!important`. I ran out of ideas but I might be missing something.